src/init/features/project.js

```diff
@@ -61,34 +62,30 @@ function _getProject(options) {
       );
     }
 
-    return prompt
-      .once({
-        type: "list",
-        name: "id",
-        message: "Select a default Firebase project for this directory:",
-        validate: function(answer) {
-          if (!_.includes(nameOptions, answer)) {
-            return "Must specify a Firebase to which you have access.";
-          }
-          return true;
-        },
-        choices: nameOptions,
-      })
-      .then(function(label) {
-        if (label === NEW_PROJECT || label === NO_PROJECT) {
-          return {
-            id: label,
-          };
+    return promptOnce({
+      type: "list",
+      name: "id",
+      message: "Select a default Firebase project for this directory:",
+      validate: function(answer) {
+        if (!_.includes(choices, answer)) {
+          return "Must specify a Firebase to which you have access.";
         }
+        return true;
```

Comment:
I'm now trying to convert this file to TS in my other PR - does this function actually return either a string or a boolean? Seems....odd...